greymatter/values.yaml

```
@@ -1,5 +1,7 @@
 global:
   environment: openshift
+  # If `environment: kubernetes`, set to true to install Voyager ingress 
```

Should we add a little more definition to this comment?

```suggestion
  # If `environment: kubernetes`, set to true to install Voyager ingress.  If left `false`, then no ingress will be automatically installed by Helm and you will need to manually add an ingress configuration to your Kubernetes ingress controller.
```